src/content/en/fundamentals/engage-and-retain/push-notifications/permission-ux.md

```diff
@@ -0,0 +1,177 @@
+
+No context as to what the notifications are for, if the user wanted to get a
+task done (i.e. check a flight or book a flight), this permission prompt gets in
+the way of that and secondly it's not a good look having a pop-up over the site.
+
+### Double Permission
+
+You may feel that your site has a clear need for push messaging and as a result
+want to ask the user for permission as soon as possible. An example of this
+sites that fit into this category are messaging sites or email clients. You
+receiving a message or email, show a notification.
+
+In these cases, it's worth considering the double permission UX.
+
+With this approach you display a custom permission prompt in your web app which
+asks the user to enable notifications. By doing this the user can say enable or
+disable without risk of being permanently blocked. If the user selects enable on
+the custom UI, display the actual permission prompt, otherwise hide your custom
+pop-up and ask some other time.
+
+A good example of this is [Slack](https://slack.com/). They show a prompt at
+the top of their page once you've signed in asking if you'd like to enable notifications.
+
+
+
+### Settings Panel
+
+You can move notifications into a settings panel, giving users an easy way
+to enable and disable push messaging, without the need of cluttering your
+web app's UI.
+
+A good example of this is [Google I/O's 2016 site](https://events.google.com/io2016/). When you
+first load up the Google I/O site, you aren't asked to do anything,
+the user is left to explore the site.
+
+![When you first load the page, no prompt, just calm on Google IO.](./images/ux-examples/google-io/google-io-first-load.png){: .device-image .center-image }
+
+After a few visits of clicking the menu item on the right, a settings panel is reveal to the
+user, allowing them to set up and manage notifications.
+
+![Settings panel on Google IO's web app for push messaging.](./images/ux-examples/google-io/google-io-settings-panel.png){: .device-image .center-image }
+
+Clicking on the checkbox displays the permission prompt, no hidden surprises.
+
+![Google IO's web app displaying the permission prompt.](./images/ux-examples/google-io/google-io-permission-prompt.png){: .device-image .center-image }
+
+After the permission has been granted the checkbox is checked and the user is
+good to go. The great thing about this UX is that the location to sign up for
+push is the same location to disable push.
+
+
+
+### Passive Approach
+
+One of the easiest ways to offer push to a user is to have a button
+or toggle switch that enables / disables push messages in a location
+on the page that is consistent throughout a site.
+
+This doesn't drive users to enable push notifications, but consistency and allowing users to
+opt in and out easily without constant nudging gives users a way to engage with a site /
+service / brand if and when they want to. For sites like blogs that might have some regular
```

Comment:
I had trouble digesting this sentence. Try:

This doesn't drive users to enable push notifications, but consistency and user choice-allowing users to opt in and out easily without constant nudging-gives users a way to engage with a site if and when they want to.